t/unittest/dal.t

```diff
@@ -120,6 +120,27 @@ is($node->voip, "yes", "Voip was saved");
 
 pf::dal::node->remove_by_id({mac => $test_mac});
 
+$node = pf::dal::node->new({ mac => $test_mac });
```

Comment:
Can you set like computername = "zams-computer" here and validate that it will still be there when node2 gets saved.

Just to test that conflicting fields are overwritten but non-conflicting fields are untouched